compiler/loco/include/loco/IR/DataTypeTraits.h

```diff
@@ -123,6 +129,8 @@ inline uint32_t size(DataType data_type)
 {
   switch (data_type)
   {
+    case DataType::S4:
+      return sizeof(DataTypeImpl<DataType::S4>::Type);
```

Comment:
Size of int4 is 0.5 byte, but this implementation will return 1. I imagined exception will be thrown here.

Reply:
S4 for to store values in IR is to use 1 byte for simple calculation, -8 to 0 and to 7.

Reply:
I see. Since loco will also affect our backend compiler, @parjong may be related.